packages/zosfiles/src/methods/download/Download.ts

```diff
@@ -521,7 +521,15 @@ export class Download {
 
             const writeStream = options.stream ?? IO.createWriteStream(destination);
 
-            // If data type is not defined by user, check for USS tags
+            if(options.attributes)
+            {
+                options.binary = options.attributes.getFileTransferMode(ussFileName, options.binary) === TransferMode.BINARY;
+                const remoteEncoding = options.attributes.getRemoteEncoding(ussFileName);
+                if(remoteEncoding === Tag.BINARY) options.encoding = undefined;
```

Comment:
Maybe I am not seeing the big picture here, but does options.binary get set if the remote encoding is Binary? If not, could this result in a file that is tagged binary being uploaded in IBM-1047?

Reply:
Disregard, I thought this was the tagged encoding.